components/Tabs/ivanmora-solution.js

```diff
@@ -0,0 +1,78 @@
+class TabItem {
+    constructor(element) {
```

Comment:
Keep an eye for consistent spacing. I notice 4 space indentation here vs 2 spaces on line 3.